src/Input/AbstractParamAwareInput.php

```diff
@@ -288,11 +288,10 @@ private function askQuestion(Question $question, bool $valueIsArray, bool $value
 
             if ($valueIsRequired && [] === $values) {
                 $question->setValidator(
-                    /**
-                     * @psalm-template ValueType of mixed
```

Comment:
Why were these reverted?

Reply:
because template annotations for closures are not supported. 
It seems that they were ignored in older psalm versions.